test/check-machines-settings

```diff
@@ -705,6 +710,28 @@ class TestMachinesSettings(machineslib.VirtualMachinesCase):
         b.wait_visible("#watchdog-dialog-apply[aria-disabled=true]")
         b.mouse("#watchdog-dialog-apply", "mouseenter")
         b.wait_visible("#watchdog-live-edit-tooltip")
+        b.click("#vm-subVmTest2-watchdog-modal button[aria-label=Close]")
+        b.wait_not_present("#vm-subVmTest2-watchdog-modal")
+
+        self.goToMainPage()
+        self.waitPageInit()
+
+        m.execute("virsh destroy subVmTest2")
+
+        # Create a Q35 machine, which has a permanent watchdog that
+        # can't be removed.  (Some older versions of libvirt don't add
+        # a watchdog.)
+
+        m.execute("virsh destroy subVmTest1; virsh undefine subVmTest1")
+        args = self.createVm("subVmTest1", os="linux2022")
+        has_watchdog = m.image not in ["debian-stable", "ubuntu-2204", "rhel-8-10"]
+
+        self.goToVmPage("subVmTest1")
+        if has_watchdog:
+            b.wait_in_text("#vm-subVmTest1-watchdog-state", "Reset")
+            setWatchdogActionLive(action="pause", previous_action="reset", reboot_machine=True, removable=False)
+        else:
+            b.wait_in_text("#vm-subVmTest1-watchdog-state", "none")
```

Comment:
This is still wrong, like in the previous thread. The [pixel tests](https://github.com/cockpit-project/pixel-test-reference/compare/2e5564298318492cda67adbcffc3fe1e06db2fa0..31e5a6717ab7d31db535e983efc725352fb361b9) clearly show that the "none" gets humanized to "Do nothing", in the dialog. This check looks at the VM details page (not the dialog), and wants "none". So something is still wrong with "translating" the API value to the string in this case. It clearly works for "Reset".

Reply:
In the UI "none" means "No watchdog".  "Do nothing" means "watchdog with action 'none'".

Reply:
Ah, sorry!